frontend/src/sagas.js

```diff
@@ -875,11 +891,114 @@ export function* executeConfirmedActionsSaga({ showLoading, projectId, subprojec
         })
       );
     yield put({
-      type: EXECUTE_CONFIRMED_ACTIONS_SUCCESS
+      type: EXECUTE_CONFIRMED_ADDITIONAL_ACTIONS_SUCCESS
     });
   }, showLoading);
 }
 
+export function* executeConfirmedPostActionsSaga({ actions, showLoading, projectId, subprojectId, workflowitemId }) {
+  yield execute(function*() {
+    yield put({
+      type: EXECUTE_CONFIRMED_POST_ACTIONS
+    });
+
+    const permissionsChange = {
+      projectId: "",
+      subprojectId: "",
+      workflowitemId: ""
+    };
+    const postActions = actions.map(action => {
+      action.id = workflowitemId;
+      return action;
+    });
+
+    for (const index in postActions) {
+      const action = postActions[index];
+      try {
+        yield* executeConfirmedAction(action, permissionsChange, projectId, subprojectId);
```

Comment:
Do not mutate the object in the function and use it afterwards.

Reply:
done